cmd/plugin/main.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 func main() {
-	var rootCmd = &cobra.Command{
+	rootCmd := &cobra.Command{
```

Comment:
Can you rename this?

It's called `rootCmd`, but it's not the actual root command anymore. What about `exampleCmd`?

Reply:
done